lib/azure/storage/blob/blob_service.rb

```diff
@@ -45,25 +45,7 @@ def initialize(options = {})
       end
       
       def call(method, uri, body=nil, headers={})
-        # Force the request.body to the content encoding of specified in the header
```

Comment:
Deleting this block will break the scenarios of force encoding the body with the specified charset. Using "Content-Encoding" is wrong, we are going to use the "charset" attribute in the content-type.